deployment/helm/skaha/templates/session-volumes-mounts.yaml

```diff
@@ -0,0 +1,52 @@
+{{- define "skaha.session.commonVolumeMounts" -}}
+        {{- if .Values.sessions.extraVolumes }}
+        {{- range .Values.sessions.extraVolumes }}
+        - name: {{ .name }}
+          mountPath: {{ .volumeMount.mountPath }}
+          {{- if .volumeMount.subPath }}
+          subPath: {{ .volumeMount.subPath }}
+          {{- end }}
+          {{- if .volumeMount.readOnly }}
+          readOnly: {{ .volumeMount.readOnly }}
+          {{- end }}
+          {{- if .volumeMount.mountPropagation }}
+          mountPropagation: {{ .volumeMount.mountPropagation }}
+          {{- end }}
+        {{- end }}
+        {{- end }}
+        - mountPath: "/etc/passwd"
+          name: etc-passwd
+          subPath: passwd
+        - mountPath: "/etc/group"
+          name: etc-group
+          subPath: group
+        - mountPath: "${SKAHA_TLD}"
+          name: cavern-volume
+          subPath: "cavern"
+        - mountPath: "/scratch"
+          name: scratch-dir
+          subPath: "${skaha.sessionid}"
+{{- end }}
+
+{{- define "skaha.session.cartaVolumeMounts" -}}
+        - mountPath: "/skaha-system"
+          name: start-carta
+{{- end }}
```

Comment:
I don't think we need to generalize the type-specific mounts, only the common set of mounts that will go into each session (regardless of type).  So, I would suggest returning these startup script mounts back to the launch-*.yaml files.

Another reason to do that is because there are references to those scripts in the `cmd` section.  Having those in two different places creates a dependency, and thus makes it harder to understand and maintain.

Reply:
We have moved start up scripts to launch files. Thanks!